sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -19,6 +24,16 @@ export interface Key {
    * @member {string} [value] The key value.
    */
   keyMaterial?: JsonWebKey;
+  /**
+   * JsonWebKey Key Type (kty), as defined in
+   * https://tools.ietf.org/html/draft-ietf-jose-json-web-algorithms-40. Possible values include:
+   * 'EC', 'EC-HSM', 'RSA', 'RSA-HSM', 'oct'
+   */
+  keyType?: JsonWebKeyType;
+  /**
+   * Operations allowed on this key
+   */
+  keyOperations?: string[];
```

Comment:
Shouldn't this be some form of enum array? like keyOperations[], where keyOperations is either "encrypt", "decrypt", etc?

Reply:
Added